src/main/java/az/gdg/mssubscriber/controller/SubscriberController.java

```diff
@@ -0,0 +1,38 @@
+package az.gdg.mssubscriber.controller;
+
+import az.gdg.mssubscriber.model.SubscriberRequest;
+import az.gdg.mssubscriber.model.dto.SubscriberDTO;
+import az.gdg.mssubscriber.service.SubscriberService;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequestMapping("/subscriber")
+@RestController
+public class SubscriberController {
+
+    private final SubscriberService subscriberService;
+
+    public SubscriberController(SubscriberService subscriberService){
+        this.subscriberService = subscriberService;
+    }
+
+    @ApiOperation("get all subscribers")
+    @GetMapping
+    public List<SubscriberDTO> getAllSubscribers(){
+        return subscriberService.getAllSubscribers();
+    }
+
+    @ApiOperation("create subscriber")
+    @PostMapping
+    public void createSubscriber(@RequestBody SubscriberRequest subscriberRequest){
+        subscriberService.createSubscriber(subscriberRequest);
+    }
+
+    @ApiOperation("delete subscriber by id")
+    @DeleteMapping("/{id}")
```

Comment:
to delete himself/herself from subscribers list, it seems like you are asking your customer to enter his own id from table, how come a customer knows his id in table ? He only knows his email. Please delete subscriber by email, because only backend knows the ids.